docs/nidcpower/class.rst

```diff
@@ -149,6 +149,16 @@ Session
 
     :type independent_channels: bool
 
+    :param _grpc_options:
+        
+
+        MeasurementLink gRPC session options
+
+        
+
+
+    :type _grpc_options: nidcpower.grpc_session_options.GrpcSessionOptions
```

Comment:
Can we update this so that it links to the documentation for that class? Maybe this will help: https://stackoverflow.com/a/59682507

There is `:py:class:` for referencing a class.
Feel free to close if you feel it's better not to link to it for some reason.

Reply:
Made the links work.